src/imgui/ImGuiSpriteViewer.cc

```diff
@@ -459,6 +476,9 @@ void ImGuiSpriteViewer::paint(MSXMotherBoard* motherBoard)
 			auto zoomSize = float(zm * size);
 			auto fullSize = zoomSize * gl::vec2(8, 4);
 			im::Child("##attrib", {0, fullSize.y}, 0, ImGuiWindowFlags_HorizontalScrollbar, [&]{
+				auto formatColorData = [&](uint16_t address) {
+					return formatToString([&](unsigned addr){ return vram[addr]; }, address, address + size - 1, {}, 1, {}, "%02X", manager.getInterpreter());
+				};
```

Comment:
This `formatColorData()` function is similar to the `formatPatternData()` function above. Maybe we can unify them into a single function that takes both an `address` and a `size` parameter?